internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -78,9 +80,7 @@ func (r *AgentServiceConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result,
 	ctx := context.Background()
 	instance := &adiiov1alpha1.AgentServiceConfig{}
 
-	// NOTE: ignoring the Namespace that seems to get set on request when syncing on namespaced objects
-	// when our AgentServiceConfig is ClusterScoped.
-	if err := r.Get(ctx, types.NamespacedName{Name: req.NamespacedName.Name}, instance); err != nil {
```

Comment:
Hi @djzager. I didn't understand why it was previously looking up the AgentServiceConfig using name only. I changed it to NamespacedName and it also works. Am I overlooking something?